README.md

```diff
@@ -36,3 +36,55 @@ The repository is organized this way:
 
 Presently, only the `v1` version of the formats exist.
 Hence, to update the list of providers, file a pull-request against the repository to edit `/src/links/v1/providers.json`.
+
+## Fork this repository to set up an index meta-database for your own databases
+
+- Click on the Fork button in GitHub and follow the instructions.
+
+- Edit `/src/info/v1/info.json` with your information:
+
+  - Make sure to update the URL in `available_api_versions` to point at your hosting location.
+
+- Edit `/src/info/v1/info.json` to point out your OPTiMaDe databases:
```

Comment:
```suggestion
- Edit `/src/info/v1/links.json` to point out your OPTiMaDe databases:
```

Reply:
It is great to get things like these carefully proofed! Thanks!